app/main.py

```diff
@@ -1 +1,30 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(self, name: str, health: int = 100) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = False
```

Comment:
the hidden attribute is likely to be set to different values at the creation time of the object, it is better to allow it to be set via the constructor parameters and there you can add a default value False.